nixops/resources/ebs_volume.py

```diff
@@ -141,6 +141,10 @@ def create(self, defn, check, allow_reboot, allow_recreate):
                 states=['available', 'in-use'])
             self.state = self.UP
 
+    def check(self):
+        volume = nixops.ec2_utils.get_volume_by_id(self.connect(self.region), self.volume_id)
+        if volume is None:
+            self.state = self.MISSING
```

Comment:
we probably need a warning that the volume is no longer available here as well.

Reply:
Well, the disks check doesn't include notes like the machine one. But after updating the volume state, the check table show it as missing
```
Machines state:
+---------+--------+-----+-----------+----------+----------------+----------------------------------------+------------------------------------------------------------+
| Name    | Exists | Up  | Reachable | Disks OK | Load avg.      | Units                                  | Notes                                                      |
+---------+--------+-----+-----------+----------+----------------+----------------------------------------+------------------------------------------------------------+
| machine | Yes    | Yes | Yes       | No       | 0.00 0.00 0.00 | proc-sys-fs-binfmt_misc.mount [failed] | volume ‘vol-096cbe0647ee88eae’ not attached to ‘/dev/xvdf’ |
|         |        |     |           |          |                |                                        | volume ‘vol-096cbe0647ee88eae’ no longer exists            |
+---------+--------+-----+-----------+----------+----------------+----------------------------------------+------------------------------------------------------------+
Non machines resources state:
+--------------------+--------+
| Name               | Exists |
+--------------------+--------+
| data_volume        | No     |
| my-key-pair        | Yes    |
| ssh-security-group | Yes    |
+--------------------+--------+
```
The only thing that I can add is a message like this 
```
data_volume.......> Volume VOLUME_ID is missing!
``` 
Before the check tables

Reply:
I see, yeah seems the existent warnings are fine already.